doorkeeper/worker/DoorkeeperRevisionFeedWorker.php

```diff
@@ -180,30 +178,31 @@ private function sendUpdateRequest($revision, $accepted_bmo_ids) {
         ->setPath('/rest/phabbugz/update_reviewer_statuses');
 
       $future = $this->get_http_future($future_uri)
-        ->setMethod('PUT')
-        ->setData($request_data)
-        ->setExpectStatus(array(200));
+        ->setData($request_data);
 
-      $this->log(pht('Making a request to: %s', (string) $future_uri));
-      $this->log('Using data:'.json_encode($request_data, JSON_FORCE_OBJECT));
+      echo(pht('Making a request to: %s', (string) $future_uri));
+      echo('Using data:'.json_encode($request_data, JSON_FORCE_OBJECT));
 
         try {
           list($status) = $future->resolve();
           $status_code = $status->getStatusCode();
           if($status->getStatusCode() != 200) {
-            $this->log(pht('sendUpdateRequest failure: BMO returned code %s', $status_code));
+            echo(pht('sendUpdateRequest failure: BMO returned code %s', $status_code));
           }
         }
         catch(HTTPFutureResponseStatus $ex) {
           $status_code = $status->getStatusCode();
-          $this->log(pht('obsoleteAttachment exception: %s %s', $status_code, $ex->getErrorCodeDescription($status_code)));
+          echo(pht('obsoleteAttachment exception: %s %s', $status_code, $ex->getErrorCodeDescription($status_code)));
         }
     }
 
     private function get_http_future($uri) {
       return id(new HTTPSFuture((string) $uri))
         ->addHeader('X-Bugzilla-API-Key', PhabricatorEnv::getEnvConfig('bugzilla.automation_api_key'))
+        ->setMethod('PUT')
         ->addHeader('Accept', 'application/json')
+        ->addHeader('Content-Type', 'application/json')
```

Comment:
we will need to remove this as it causes the data to be passed as form-urlencoded :(